### ISSUE TITLE
PREFER_DATES_FROM future and parse("1 min") returns time 1 min ago 

### ISSUE THREAD

**Issue by raghur** [scrapinghub/dateparser#563]
```
In [13]: parse("1 min", settings={"PREFER_DATES_FROM" : "future"}) - datetime.now()                                                                                                          
Out[13]: datetime.timedelta(-1, 86339, 999551)
```
Even when 'PREFER_DATES_FROM' is set to 'future', without the 'in' keyword, the time returned in 1 min before.

Expectation is that if future dates are preferred, then `1 min` should return 1 min in future.

**Comment by Dimbreath** [comment]
I can confirm this. Any solution so far?

**Comment by Watchful1** [comment]
I submitted a pull request to fix this months ago but it was never merged: https://github.com/scrapinghub/dateparser/pull/530

**Comment by noviluni** [comment]
Hi @raghur @Dimbreath this is fixed in the last version `0.7.4` :tada:
